articles/client-platforms/_includes/_rules-create-section.md

```diff
@@ -0,0 +1,14 @@
+To create a rule just go to [rule new page](https://manage.auth0.com/#/rules/new). You can create it from scratch or use an existing template. These templates are written by Auth0 team to assist you complete common tasks. 
+Let's use the one called *'Add country to the user profile'*
+
+![Empty rule](/media/articles/rules/rule-choose-add-country-template.png)
```

Comment:
@alemata Could you please resize these 2 images as well? They are both 1212 × 853 pixels. Thank you!

- https://github.com/auth0/docs/tree/master/media/articles/rules/rule-choose-add-country-template.png
- https://github.com/auth0/docs/tree/master/media/articles/rules/rule-create-add-country-country.png